app/views/upload_your_files.scala.html

```diff
@@ -99,6 +100,13 @@
                     </br>
 
                     @govukButton(Button(content = Text(messages("fileUploadPage.button")), attributes = Map("id" -> "submit", "name" -> "continue")))
+
+                    @link(
+                        id = Some("cancel-link"),
+                        classes = "govuk-link govuk-!-margin-left-5 govuk-!-margin-top-2",
+                        text = messages("fileUploadPage.cancel"),
+                        style = Some("display: inline-block"),
```

Comment:
Are you sure that we do not have already a css class somewhere for this?

Reply:
Created a new class to accommodate the inline-block :+1: